readme.md

```diff
@@ -1,3 +1,3 @@
-# Repository for class project
+#Advanced Internet Programming Project
 
-This readme is a placeholder that should be replaced
+Meghanath Reddy.B
```

Comment:
You'll want to make sure your readme includes the steps to build your application